delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/GlobalEntityDefinition.scala

```diff
@@ -0,0 +1,14 @@
+package ch.epfl.bluebrain.nexus.delta.sourcing
+
+import ch.epfl.bluebrain.nexus.delta.sourcing.event.Event
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.EntityType
+import ch.epfl.bluebrain.nexus.delta.sourcing.state.State
+import doobie.{Get, Put}
+
+final case class GlobalEntityDefinition[Id, S <: State, Command, E <: Event, Rejection](
```

Comment:
I split the global definition apart as it never have tagging nor declare dependencies between entities